tutorials/10-bayesian-differential-equations/10_bayesian-differential-equations.jmd

```diff
@@ -73,6 +73,63 @@ To make the example more realistic we add random normally distributed noise to t
 sol = solve(prob, Tsit5(); saveat=0.1)
 odedata = Array(sol) + 0.8 * randn(size(Array(sol)))
 
+# alternatively, we can use real-world data from Hudson’s Bay Company records (a Stan implementation can be found here: https://mc-stan.org/users/documentation/case-studies/lotka-volterra-predator-prey.html)
+
+# years = [1900,1901,1902,1903,1904,1905,1906,1907,1908,1909,1910,1911,1912,1913,1914,1915,1916,1917,1918,1919,1920]
```

Comment:
We need also time points, otherwise we don't know how to solve the ODE.

Reply:
> We need also time points, otherwise we don't know how to solve the ODE.

@devmotion that's true, thanks for the reminder. I just modified `tspan` to accommodate the new data.